### PR TITLE
Add Python support to run_interop_tests.py

### LINKED ISSUE #3632: Add python support to run_interop_tests.py
This is basically a question of figuring out a new Dockerfile and build_interop.sh script, the rest is very easy. I'll be happy to provide some help.

Related PRs:
https://github.com/grpc/grpc/pull/3605
https://github.com/grpc/grpc/pull/3631

### PR DESCRIPTION
**Pull request by soltanmm**
Fixes #3632. Maybe.

Is this sufficient?

cc @nathanielmanistaatgoogle

### CODE REVIEW

**Comment by jtattermusch** [comment]
Changes are in the right direction, but it looks like all the python runs fail to locate libgrpc.so.

My guess is you are either missing a make install_?  step or you're not setting LD_LIBRARY_PATH properly. What does run_tests.py do to be able to locate libgrpc?

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/var/local/git/grpc/python2.7_virtual_environment/lib/python2.7/site-packages/grpc_interop/client.py", line 35, in <module>
    from grpc.early_adopter import implementations
  File "/var/local/git/grpc/python2.7_virtual_environment/local/lib/python2.7/site-packages/grpc/early_adopter/implementations.py", line 34, in <module>
    from grpc._adapter import fore as _fore
  File "/var/local/git/grpc/python2.7_virtual_environment/local/lib/python2.7/site-packages/grpc/_adapter/fore.py", line 38, in <module>
    from grpc._adapter import _intermediary_low as _low
  File "/var/local/git/grpc/python2.7_virtual_environment/local/lib/python2.7/site-packages/grpc/_adapter/_intermediary_low.py", line 57, in <module>
    from grpc._adapter import _low
  File "/var/local/git/grpc/python2.7_virtual_environment/local/lib/python2.7/site-packages/grpc/_adapter/_low.py", line 31, in <module>
    from grpc._adapter import _c
ImportError: libgrpc.so.0: cannot open shared object file: No such file or directory

FLAKE: cloud_to_cloud:python:go_server:empty_stream [ret=1, pid=16614]
```

**Comment by soltanmm** [comment]
Should be running and working fine now, sans an unimplemented interop test (cc #3807).

**Comment by jtattermusch** [comment]
```
Traceback (most recent call last):
  File "tools/run_tests/run_interop_tests.py", line 598, in <module>
    docker_image=docker_images.get(str(language)))
  File "tools/run_tests/run_interop_tests.py", line 388, in cloud_to_prod_jobspec
    environ = dict(language.cloud_to_prod_env(), **language.global_env())
TypeError: type object argument after ** must be a mapping, not NoneType
```

File: tools/run_tests/run_interop_tests.py [hunk 1]
```diff
@@ -139,6 +145,9 @@ def cloud_to_prod_env(self):
   def server_args(self):
     return ['./run-test-server.sh', '--use_tls=true']
 
+  def global_env(self):
+    pass
```
Comment (soltanmm) [inline tools/run_tests/run_interop_tests.py]: Note to self: this is wrong.

**Comment by soltanmm** [comment]
Is there a word for 'silly-and-obvious-mistake-referenced-in-a-comment-on-an-outdated-commit'? If so, <that word here> is fixed.

**Comment by jtattermusch** [comment]
Thanks!

**Pull request merged**